presto-benchmark/src/test/java/com/facebook/presto/benchmark/MemoryLocalQueryRunner.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.execution.TaskId;
+import com.facebook.presto.execution.TaskStateMachine;
+import com.facebook.presto.memory.MemoryPool;
+import com.facebook.presto.memory.QueryContext;
+import com.facebook.presto.operator.Driver;
+import com.facebook.presto.operator.TaskContext;
+import com.facebook.presto.plugin.memory.MemoryConnectorFactory;
+import com.facebook.presto.spi.QueryId;
+import com.facebook.presto.spi.memory.MemoryPoolId;
+import com.facebook.presto.testing.LocalQueryRunner;
+import com.facebook.presto.testing.NullOutputOperator;
+import com.facebook.presto.tpch.TpchConnectorFactory;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.units.DataSize;
+import org.intellij.lang.annotations.Language;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static io.airlift.units.DataSize.Unit.GIGABYTE;
+import static io.airlift.units.DataSize.Unit.MEGABYTE;
+
+public class MemoryLocalQueryRunner
+{
+    protected LocalQueryRunner localQueryRunner = createMemoryLocalQueryRunner();
+
+    public void execute(@Language("SQL") String query)
```

Comment:
`LocaleQueryRunner::execute` materializes output, which can skew benchmark results. This one, uses `NullOutputFactory`.